src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -432,6 +438,10 @@ protected int doWork() {
         if (SECOND_INPUT_SAMPLE_MAP != null) {
             IOUtil.assertFileIsReadable(SECOND_INPUT_SAMPLE_MAP);
         }
+        if (SAMPLE_INDIVIDUAL_MAP != null) {
+            IOUtil.assertFileIsReadable(SAMPLE_INDIVIDUAL_MAP);
+            sampleIndividualMap = getStringStringMap(SAMPLE_INDIVIDUAL_MAP, "SAMPLE_INDIVIDUAL_MAP");
```

Comment:
I would emit a warning if any of the individuals are also unmapped samples, as that might not be intended.

Reply:
Also, perhaps emit a warning if any of the samples in the map are not samples in the input.

Reply:
added warnings